common/psbt_keypath.c

```diff
@@ -0,0 +1,33 @@
+#include "config.h"
+#include <bitcoin/tx.h>
+#include <common/psbt_keypath.h>
+#include <common/utils.h>
+#include <wally_bip32.h>
+#include <wally_psbt.h>
+
+void psbt_set_keypath(u32 index, const struct ext_key *ext, struct wally_map *map_in) {
+	u8 fingerprint[BIP32_KEY_FINGERPRINT_LEN];
+	if (bip32_key_get_fingerprint(
+		    (struct ext_key *) ext, fingerprint, sizeof(fingerprint)) != WALLY_OK)
+		abort();
```

Comment:
Would it make sense to use `fatal("some sensible message")` here?

Reply:
`fatal` is `undefined reference` when this gets linked into `lightning_{channeld,closingd,onchaind}`.  I don't find any uses of `fatal` in the daemons, mostly `abort`.  I could ...
1. return wally status from these routines and let the callers `abort` or `fatal` depending on context
2. link something into the daemons to provide `fatal`
3. print something to `stderr` and `abort`
4 ...?
5. Profit!

Thoughts?

Reply:
That's quite alright, I shouldn't review code while on a plane :-)